### ISSUE TITLE
dimStyle.DIMTXSTY It would be better to provide a reasonable default value.

### ISSUE THREAD

**Issue by FishOrBear** [dxfjs/writer#74]
```
dimStyle.DIMTXSTY = "16";//Equal to the default Dimstyle handle. Without it, the annotation style will not be correctly displayed.
```

what do you think

**Comment by tarikjabiri** [comment]
yeah but not sure if it will be convenient for everyone, thus the DIMSTYLE is customizable one can set the values they need.

**Comment by tarikjabiri** [comment]
This is set to standard style handle by default, not sure what is the problem can you provide an example so I can see the problem

**Comment by FishOrBear** [comment]
[dxf.zip](https://github.com/dxfjs/writer/files/11284375/dxf.zip)

![image](https://user-images.githubusercontent.com/19372111/233316759-615233b5-f5a5-4a15-b6df-ee142ea1077f.png)

![image](https://user-images.githubusercontent.com/19372111/233316796-4deb13da-f7be-4c76-ba89-ad4bd25191cf.png)

This is the dxf file I generated. The only difference between them is the default style. When no default inherited style is specified, the dimension text disappears.

Because the dxf we write may not provide all dimstyle values, it would be better to give the default inherited style.

**Comment by tarikjabiri** [comment]
Using the latest version 2.8.1 you do it like this

```js
// Customize the dimStyleStandard
dxf.document.dimStyleStandard.DIMTXT = 16 // The text height
const dim = dxf.addLinearDim(point3d(), point3d(200, 0), {
  offset: 10,
  styleName: dxf.document.dimStyleStandard.name
});
```

Also you can use the header variables to customize the look of the dimensions
```js
dxf.setVariable('$DIMTXT', { 40: 10 }); // The text height
dxf.setVariable('$DIMASZ', { 40: 10 }); // Dimensioning arrow size
```
